src/services/doppler-legacy-client.ts

```diff
@@ -689,6 +703,27 @@ export class HttpDopplerLegacyClient implements DopplerLegacyClient {
     }
   }
 
+  public async requestAgenciesDemo(
+    model: RequestAgenciesDemoModel,
+  ): Promise<RequestAgenciesDemoResult> {
+    const response = await this.axios.post('/WebApp/RequestAgenciesDemo', {
+      Email: model.email,
+      Firstname: model.firstname,
+      Lastname: model.lastname,
+      Phone: model.phone,
+      ContactSchedule: model.range_time,
+      SendingVolume: model.volume,
+    });
+
+    if (!response.data.success) {
```

Comment:
I have a doubt here because if something wrong happens in BE (a problem with the api or a problem sending the email) I won't show the success message (and we don't have an error message neither). 
It doesn't really matters if, for example, the subscriber isn't added to the Doppler account list but we send the email anyway. 
It won't be nice to show a success message always... or yes? 🤔 

My doubt is: when I should show the success message?

Reply:
Another thing... what happens if the email that I want to register is in a black list? It's ok to send the email to unbounce in that case? 🤔

Reply:
I'm not sure if the email is in black list how the process will be handled, perhaps @javierburger will know. Is there an interceptor for this cases or should we look into the blacklist? Either way this is not done here but in old doppler.

Reply:
> It doesn't really matters if, for example, the subscriber isn't added to the Doppler account list but we send the email anyway.

I would say in most cases like this, like in reset password we will always tell the user that all is ok. But it may fail.

Reply:
Ok, it won't be normal to fail in both actions, so well, I will always show the success message (in the worst case the user won't be never called 😋)